src/style.css

```
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bigInput {
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px;
+  background-size: 17px;
+  padding-left: 58px;
+  margin-top: 12px;
+}
+
+.bigInput::placeholder {
+  position: absolute;
+  top: 25px;
+  right: 709px;
+  bottom: 23px;
+  left: 62px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 21.86px;
+}
```

you don't need add to your placeholder `position: absolut`
```suggestion
.bigInput::placeholder {
  font-family: Avenir, sans-serif;
  font-size: 16px;
  line-height: 22px;
}
```